src/backend/distributed/operations/shard_rebalancer.c

```diff
@@ -1668,7 +1674,9 @@ MoveShardCost(NodeFillState *sourceFillState,
  * it couldn't.
  */
 static bool
-FindAndMoveShardCost(float4 utilizationLowerBound, float4 utilizationUpperBound,
+FindAndMoveShardCost(float4 utilizationLowerBound,
```

Comment:
why is this named `FindAndMoveShardCost` but not something like `NextShardMoveCost`?

Reply:
Because it's finding and moving a `ShardCost`.